pyvista/plotting/renderer.py

```diff
@@ -1537,9 +1541,7 @@ def hide_axes(self) -> None:
         >>> pl.hide_axes()
 
         """
-        if hasattr(self, 'axes_widget') and self.axes_widget.GetEnabled():
-            self.axes_widget.EnabledOff()
-            self.Modified()
+        self.delete_axes_widget()
```

Comment:
This substitution is incorrect I think... don't want to delete the axes in this case just hide them so that `show_axes` can be called later